src/components/video-card/video-card.css

```diff
@@ -0,0 +1,34 @@
+.video-card-container {
+  display: flex;
+  flex-direction: column;
+  width: 17.5rem;
+  box-shadow: 0 0 3px var(--color-silver);
+  background-color: var(--color-white);
+}
+
+.video-card-container > i {
+  font-size: 1.5rem;
+  color: var(--color-gray);
+  position: absolute;
+  top: 10px;
+  right: 12px;
```

Comment:
you can use rem here.

Reply:
I used px because it is less than 16px. Vikram said it will make not much difference using rems when it is less than 16px.